### ISSUE TITLE
TeamServer error -> Segmentation fault

### ISSUE THREAD

**Issue by BabelQwerty** [maxDcb/C2TeamServer#2]
error log

```
$ ./TeamServer
[2024-05-14 07:00:57.750875] [0x000070e86ca98cc0] [info]    Team Server listening on 0.0.0.0:4444
[2024-05-14 07:01:27.675115] [0x000070e868bf8640] [info]    AddListener Tcp 0.0.0.0:443
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - sendData: sendData
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - initServer: initServer
SocketHandler.cpp - receive: receive
SocketHandler.cpp - receive: receive failed: End of file
SocketHandler.cpp - closeConnection: closeConnection
SocketHandler.cpp - closeConnection: closeConnection
Segmentation fault (core dumped)
```

**Comment by maxDcb** [comment]
I think I found the issue! 

I am not setting 'm_socketTcp=nullptr;' int SocketHandler.cpp : Server::closeConnection() after I deleted it!
![image](https://github.com/maxDcb/C2TeamServer/assets/40819564/21a961f6-6d7f-4a21-a44a-9991c36f0b8e)

Should be fixed in the 0.0.2

Thanks for the report!

**Comment by BabelQwerty** [comment]
Fixed ,thanks.
